.github/workflows/ci-cd-default.yml

```
@@ -204,3 +204,16 @@ jobs:
         with:
           dsb-build-envs-array: ${{ steps.read-build-envs.outputs.json-array }}
           app-config-repo-token: ${{ secrets.app-config-repo-token }}
+
+      #tag git commit
+      - uses: actions/github-script@v6
+        env:
+          tag: ${{ format('refs/tags/{0}', needs.create-matrix.outputs.applications-version) }}
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: process.env.tag,
```

Kan du ikke gjøre sånn direkte?
```suggestion
              ref: 'refs/tags/${{ needs.create-matrix.outputs.applications-version) }}',
```

---

Det kan du ha rett i . Jeg endret det etter første kjøring som feilet men det var nok pga at det var feil variabel som ble lest inn.